### ISSUE TITLE
Platform UI - Critical and High Alerts

### ISSUE THREAD

**Issue by akachru-github** [icgc-argo/platform-ui#2208]
![image.png](https://images.zenhubusercontent.com/5d55b1b74d335f147c3e6dc0/2b2e0be6-647f-4a18-86fd-6239f76e4edf)

**Comment by akachru-github** [comment]
This may result in updating React of NextJS. That will be a separate ticket if required.

**Comment by b-f-chan** [comment]
@samrichca to complete these alerts before moving onto RNA-Seq

**Comment by samrichca** [comment]
We need to update Next from 9 to 12. Blocked by storybook.

**Comment by samrichca** [comment]
Order of tickets:
- Remove Storybook https://github.com/icgc-argo/platform-ui/issues/2297
- General upgrades https://github.com/icgc-argo/platform-ui/issues/2299
- Upgrade Apollo https://github.com/icgc-argo/platform-ui/issues/2301

**Comment by b-f-chan** [comment]
@samrichca how do you want to handle testing for this?  Do you need a BA to do a smoke test after the whole epic is done and deployed to QA?

**Comment by b-f-chan** [comment]
Yes need BA smoke test --> Wait 'til epic done!
